test.py

```diff
@@ -19,40 +22,54 @@ def download_to(url, target):
     os.rename(download_path, target)
 
 
+def validate_json(dir_path, name, fails, hubmap_schema):
```

Comment:
I'm not usually a fan of "returning" data from a function via giving a mutable data structure as input and letting the function modify it, but that can be appropriate at times. I'm not really convinced that this is one of the times in which this is necessary.

Reply:
Since this is merged, I filed new issue. #23